sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/__init__.py

```diff
@@ -35,8 +34,8 @@ class BaseExporter:
         options: :doc:`export.options` to allow configuration for the exporter
     """
 
-    def __init__(self, **options):
-        options = ExporterOptions(**options)
+    def __init__(self, **kwargs):
```

Comment:
Why do this change? Why not just utilize `BaseObject`? That way users will know which parameters are available.

Reply:
Timeout configuration is related to the specific timeout in the Http connection, we need to use ConnectionConfiguration used by the Azure client pipeline transport
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/configuration.py
We need to come back and look at all policy configurations including Retry, we need to retry more often most likely in our exporter

Reply:
BaseObject still being used I just renamed the parameter, are you asking to include an ExporterOptions parameter as well?

Reply:
Oh right, it's fine my mistake.